test/ntp_client_test.rb

```diff
@@ -646,49 +637,33 @@
 
     it "sets modified as true if deleted record" do
       subject.modified = false
-
       subject.deleteSyncRecord(3)
-
       expect(subject.modified).to eql(true)
     end
 
     it "removes record entry from ntp records at given index position" do
-      subject.selectSyncRecord(9)
-      expect(subject.selected_record).to eql(record)
-      expect(subject.ntp_records.size).to eql(12)
-
-      expect(subject.deleteSyncRecord(9)).to eql(true)
+      expect(subject.deleteSyncRecord(0)).to eql(true)
 
-      subject.selectSyncRecord(9)
-      expect(subject.selected_record).not_to eql(record)
-      expect(subject.ntp_records.size).to eql(11)
+      subject.selectSyncRecord(0)
+      record = subject.selected_record.reject { |k| k == "cfa_record" }
+      expect(record).not_to eql(deleted_record)
+      expect(subject.ntp_records.size).to eql(5)
     end
-
   end
 
   describe "#ProcessNtpConf" do
-    let(:data_dir) { File.join(File.dirname(__FILE__), "data") }
-
-    around do |example|
-      ::FileUtils.cp(File.join(data_dir, "scr_root/etc/ntp.conf.original"),
-        File.join(data_dir, "scr_root/etc/ntp.conf"))
-      change_scr_root(File.join(data_dir, "scr_root"), &example)
-      ::FileUtils.rm(File.join(data_dir, "scr_root/etc/ntp.conf"))
-    end
-
     before do
+      allow(CFA::NtpConf).to receive(:new).and_return(ntp_conf)
       subject.instance_variable_set(:@config_has_been_read, false)
     end
 
     it "returns false if config has been read previously" do
       subject.instance_variable_set(:@config_has_been_read, true)
-
       expect(subject.ProcessNtpConf).to eql(false)
     end
 
     it "returns false if config doesn't exist" do
-      allow(Yast::FileUtils).to receive(:Exists).with("/etc/ntp.conf").and_return(false)
-
+      allow(File).to receive(:exist?).and_return(false)
```

Comment:
difference is that original FileUtils in insts-sys checks in installation environment when file exist on target disk